### ISSUE TITLE
Change Favicon 

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#59]
## User story

1. As a facility manager
2. I want / need to see the app's favicon in the opened tab
3. So that I can recognise it accordingly

## Acceptance criteria

- [ ] The app logo needs to be set as favicon in the frontend project

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release

**Comment by sn0wcat** [comment]
use siemens.ico

![favicon](https://github.com/amosproj/amos2024ss01-xcelerator-demo-app/assets/1426617/ca4bb412-f78c-4c40-8bec-5cb9dcc1b66f)
